image/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN yum -y --setopt=tsflags=nodocs install \
         google-compute-engine \
         google-compute-engine-init \
         google-config \
+        selinux-policy selinux-policy-targeted setools-console \
```

Comment:
`setools-console` package is probably not needed in the prod. version of the image, but that's fine for now.